packages/cli/lib/utils.js

```diff
@@ -547,6 +547,14 @@ exports.stringifyModelSettings = function(modelSettings) {
   );
 };
 
+exports.stringifyOptions = function(modelSettings = {}) {
+  return stringifyObject(modelSettings, {
+    indent: '  ', // two spaces
+    singleQuotes: true,
+    inlineCharacterLimit: Infinity,
+  });
+};
+
```

Comment:
Please move this utility to packages/repository-json-schema.

Reply:
I moved stringifyOptions utils to build-schema.ts (I thing there's not enough code to create its own utils file)